packages/liveblocks-redux/src/index.ts

```diff
@@ -50,15 +52,14 @@ type LiveblocksContext<
    */
   readonly isStorageLoading: boolean;
   /**
-   * Connection state of the room
+   * @deprecated Old-style connection state of the room. Prefer using the newer
+   * connection statuses.
```

Comment:
Should we tell people the new API is at `room.status`?

Reply:
Yes we should! Fixed in b2fd48220a71d45d21f2420a21d7eeccbe322c3f.